gulpfile.js

```diff
@@ -10,46 +12,51 @@ const mergeStream = require('merge-stream');
 const polymerBuild = require('polymer-build');
 const browserSync = require('browser-sync').create();
 const history = require('connect-history-api-fallback');
+const eslint = require('gulp-eslint');
+const friendlyFormatter = require('eslint-friendly-formatter');
 
 const HtmlSplitter = polymerBuild.HtmlSplitter;
 const PolymerProject = polymerBuild.PolymerProject;
 const uglify = composer(uglifyes, console);
 
-const logging = require('plylog');
+// const logging = require('plylog');
 // logging.setVerbose();
 
 const config = {
   polymerJsonPath: './polymer.json',
   build: {
-    rootDirectory: 'build'
+    rootDirectory: 'build',
```

Comment:
eslint requires comma after last property in object? What a shit..

Reply:
Airbnb and Google styleguides promote this + here is a good article about benefits: https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8